charms/kfp-profile-controller/tests/unit/test_operator.py

```diff
@@ -19,6 +21,13 @@
     KfpProfileControllerOperator,
 )
 
+# Load the custom images from the JSON
+CUSTOM_IMAGES_PATH = (
+    Path(__file__).resolve().parent.parent.parent / "src" / "default-custom-images.json"
```

Comment:
I think you don't need the `.resolve().parent.parent.parent`. Here's how we load the metadata file in some other tests https://github.com/canonical/kfp-operators/blob/2a72102ea9f42f99f60983afd8759a2de3cf215e/charms/kfp-api/tests/integration/test_charm.py#L22.